stable/xray/values.yaml

```diff
@@ -344,6 +346,34 @@ indexer:
   service:
     type: ClusterIP
 
+  ## Add custom init containers execution before predefined init containers
+  customInitContainersBegin: |
+  #  - name: "custom-setup"
```

Comment:
Was this added on purpose? I don’t see that you used it in this PR, or maybe I’m missing something?

Reply:
I added the wrong custom "thing", copy/paste fail, resolved!